### DISCUSSION TITLE
Como utilizar o CSS Grid Layout de uma maneira escalável para construir um sistema web com vários layouts diferentes?

### DISCUSSION THREAD

**Discussion by renedocket** [frontendbr/forum#1440] [Discussão]
Olá, Galera! Tudo bem com vocês?

Então, estou com uma duvida que recentemente está permeando já faz algum tempo em relação ao CSS grid layout. Por exemplo:

  Eu gostaria de fazer meu sistema de Grid por exemplo igual o do bootstrap porém com CSS Grid.

Mas eu fico meio confuso em relação a ter que criar um Grid para cada HTML novo que eu fizer quando falamos de um sistema web com diversas páginas com layouts diferentes.

Eu queria saber se vocês em algum momento já criaram algum grid especifico para algum produto em que trabalharam e quais dicas vocês aconselham em relação a isso?

Obrigado, galera! :)

**Comment by tarciozemel** [comment]
Aí é que está: com CSS Grid, _você não precisa fazer algo parecido com Bootstrap_...

A definição da grid se dá de antemão; você controla (bidimensionalmente) como será a disposição das células mesmo antes de haver conteúdo.

https://desenvolvimentoparaweb.com/css/css-grid-melhor-que-bootstrap/

**Comment by dudouz** [comment]
Fala @renedocket, esse é um assunto riquíssimo e cheio de nuances...as possibilidades e caminhos para se tomar são inúmeras. Se tu manja um pouco de inglês eu recomendo começar pela Jen Simmons, inclusive tem alguns vídeos dela que são bem ilustrativos mesmo sem o inglês pode ser que já sejam bem esclarecedores. [google.com/search?q=jen+simmons+css+grid](https://google.com/search?q=jen+simmons+css+grid).
